action/rlp_tx.go

```diff
@@ -15,7 +15,8 @@ import (
 	"golang.org/x/crypto/sha3"
 )
 
-type rlpTransaction interface {
+// RlpTransaction is an interface which makes native action compatible with eth tx
+type RlpTransaction interface {
```

Comment:
will this be used outside of action package? if not, no need to change?

Reply:
https://github.com/iotexproject/iotex-core/pull/3070/files#diff-1ee1383260f91d9a4082df0e44f3e1be95e37a8930d01712b65f7e8756b1d0e9R217